.github/workflows/beam_PostCommit_Javadoc.yml

```diff
@@ -74,6 +74,8 @@ jobs:
           github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
       - name: Setup environment
         uses: ./.github/actions/setup-environment-action
+        with:
+          java-version: 21
```

Comment:
Due to some Beam module requires higher java version, run `./gradlew :sdks:java:javadoc:aggregateJavadoc` with Java11 fails with

```
  bad class file: /Users/yathu/.gradle/caches/modules-2/files-2.1/io.debezium/debezium-core/3.1.1.Final/44cc4060846bce89761529b040accf2625e3dc0e/debezium-core-3.1.1.Final.jar(/io/debezium/relational/history/AbstractSchemaHistory.class)
    class file has wrong version 61.0, should be 55.0
```